### PR TITLE
Grenade Belts/Webbings No Longer Susceptible to Chain Reactions

### PR DESCRIPTION
**Pull request by Ophanimdev**
Your precious grenades won't blow up if they're inside grenade webbings/belts or security belts anymore.

Downsides: You can't troll by suicide bombing with these unless you move your grenades to your non-grenade safe backpacks first.

### CODE REVIEW

File: Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -505,6 +505,8 @@
           - SmokeOnTrigger
     sprite: Clothing/Belt/belt_overlay.rsi
   - type: Appearance
+  - type: ExplosionResistance
+      damageCoefficient: 0.1
```
Comment (MLGTASTICa) [inline Entities/Clothing/Belt/belts.yml]: change this to be 0.5 so strong enough explosion still trigger them

**Review by MLGTASTICa — CHANGES_REQUESTED** [review]

File: Entities/Clothing/Belt/belts.yml (hunk 1 above)
Comment (a-person5660) [reply Entities/Clothing/Belt/belts.yml]: 0.25 would be more appropriate for that, 0.5 will make it go off at pretty much any explosion iirc.

Comment (Ophanimdev) [reply Entities/Clothing/Belt/belts.yml]: 'tis done my liege.

**Comment by Ophanimdev** [comment]
It's done my liege.

**Pull request merged**